src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -145,6 +145,9 @@ class JesBackend extends Backend with LazyLogging {
     )
   }
 
+  override def engineFunctions(context: TaskExecutionContext): WdlStandardLibraryFunctions = ???
```

Comment:
I forgot about them until now.  I can do that, but I'll need to sit down with somebody to help me get JES set up (sorry, I should have been there last time but I thought it'd take longer than it did)

This would be a good exercise too because these will most likely be needed to implement functions like `write_lines`